projects/storefrontlib/src/shared/components/item-counter/item-counter.component.ts

```diff
@@ -101,7 +101,7 @@ export class ItemCounterComponent
    * If value is too small it will be set to min, if is too big it will be set to max.
    */
   adjustValueInRange(incomingValue: number): number {
-    return incomingValue < this.min || !this.min
+    return incomingValue < this.min
```

Comment:
I'm not sure I understand this change.

Reply:
`!this.min` is `true` when `min=0`. So, if `min=0`, the `value` is always `0`.